src/index.html

```diff
@@ -20,8 +20,340 @@
+        >
+          BUY
+        </a>
+      </div>
+      <div class="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="card__img"
+        />
+        <p class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__code">Product code: 195434</p>
+        <div class="container card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="reviews-text">Reviews: 5</p>
+        </div>
+        <div class="container card__price">
+          <p class="price-text">Price:</p>
+          <p class="price-value">$2,199</p>
+        </div>
+        <a
+          class="card__button"
+          data-qa="card-hover"
+        >
+          BUY
+        </a>
+      </div>
+      <div class="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="card__img"
+        />
+        <p class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__code">Product code: 195434</p>
+        <div class="container card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="reviews-text">Reviews: 5</p>
+        </div>
+        <div class="container card__price">
+          <p class="price-text">Price:</p>
+          <p class="price-value">$2,199</p>
+        </div>
+        <a
+          class="card__button"
+          data-qa="card-hover"
+        >
+          BUY
+        </a>
+      </div>
+      <div class="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="card__img"
+        />
+        <p class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__code">Product code: 195434</p>
+        <div class="container card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="reviews-text">Reviews: 5</p>
+        </div>
+        <div class="container card__price">
+          <p class="price-text">Price:</p>
+          <p class="price-value">$2,199</p>
+        </div>
+        <a
+          class="card__button"
+          data-qa="card-hover"
+        >
+          BUY
+        </a>
```

Comment:
The 'a' tag used here for the 'BUY' button does not have an 'href' attribute. If this is intended to be a button, consider using a 'button' element instead, or provide a valid 'href' if it should be a link.